### DISCUSSION TITLE
Channel names of (Homematic IP Wired Eingangsmodul - 32-fach) supported ?

### DISCUSSION THREAD

**Discussion by raphael-hettich** [SukramJ/aiohomematic#1912] [Q&A] [answered]
I am using a "Homematic IP Wired Eingangsmodul - 32-fach" and in my RaspberryMatic I have the channel individually named:

![image](https://github.com/user-attachments/assets/6cc9df5a-add4-481d-90f0-d2a244f1938f)

however in the Homeassistant integration the channels are simply called:

![image](https://github.com/user-attachments/assets/2ffde5ed-30f1-4f08-a388-88c53628159c)

is there a setting I am missing or is this simply yet unsupported?

**Comment by SukramJ** [comment]
Try [Option 3](https://github.com/danielperna84/custom_homematic#rename-of-devicechannel-in-ccu-not-reflected-in-home-assistant)

**Reply by raphael-hettich** [reply]
> Try [Option 3](https://github.com/danielperna84/custom_homematic#rename-of-devicechannel-in-ccu-not-reflected-in-home-assistant)

Thanks Suram, I tried deleting the device and restarted Homeassitant as a whole. The device came back, however the channel names stayed the same (meaning only ...chN) was reflected in the entity.

Any other ideas ?

**Reply by SukramJ** [reply]
Which version do you use?

**Reply by raphael-hettich** [reply]
> Which version do you use?

My Homeassitant:

![image](https://github.com/user-attachments/assets/6989b795-aeb7-4b53-9ed3-f969eed303ba)


My RaspberryMatic: 3.79.6.20241122

**Reply by SukramJ** [reply]
.. of the integration.

**Reply by raphael-hettich** [reply]
> .. of the integration.

![image](https://github.com/user-attachments/assets/9bb58feb-6aea-4b63-aabd-1b9aab24c769)

**Reply by SukramJ** [reply, marked as answer]
Wenn du jeweils :5 , :6, :7, :8 weg nimmst wird der Name übernommen.

**Reply by raphael-hettich** [reply]
> Wenn du jeweils :5 , :6, :7, :8 weg nimmst wird der Name übernommen.

Vielen Dank! Das hat geklappt. Ist das ein Bug?

**Reply by SukramJ** [reply]
Nein. 
Durch die verschiedenen Namensebenen (Gerät/Kanal/Datenpunkt) ist die Namensgebung recht komplex, und um das ganze dann an HA anzupassen muss man Kompromisse machen.

**Reply by raphael-hettich** [reply]
> Nein. Durch die verschiedenen Namensebenen (Gerät/Kanal/Datenpunkt) ist die Namensgebung recht komplex, und um das ganze dann an HA anzupassen muss man Kompromisse machen.

Verstehe, trotzdem vielen Dank. Ggf. macht es ja Sinn dies als Hinweis mit in das Readme aufzunehmen :-)

**Reply by SukramJ** [reply]
Kannst Du gerne machen.
